inst/templates/tiltWorkflows.Rmd

```diff
@@ -1,236 +1,207 @@
-
-# matches_mapper
-mapper_ep_ei <- read_csv("/path/mapper_ep_ei.csv")
+library(fs)
+library(tiltWorkflows)
 
-# ecoinvent_inputs
-ei_input <- read_csv("/path/ei_input_data.csv") |>
-  select("input_activity_uuid_product_uuid", "exchange_name", "exchange_unit_name") |>
-  distinct()
+# Helper
+tilt_path <- function(...) {
+  path_home("Downloads", "tilt", ...)
+}
 
-# isic_4digit_name
-isic_4digit_name <- read_csv("/path/isic_4digit_name.csv")
+dir_create(tilt_path())
+options(width = 500)
 ```
 
-## Emissions profile
-
-### Load Data
+## Data
 
-The input data of Emissions_profile which comes from tiltIndicatorBefore package
-becomes the input for the tiltIndicator package.
+Here we use toy datasets that you should replace with your real data.
 
 ```{r}
-# emissions_profile companies
-ep_company <- read_csv("/path/emissions_profile_any_companies_ecoinvent.csv")
+europages_campanies <- tiltIndicatorAfter::ep_companies |>
+  select(
+    "company_name",
+    "country",
+    "company_city",
+    "postcode",
+    "address",
+    "main_activity",
+    "companies_id"
+  )
+
+ecoinvent_activities <- tiltIndicatorAfter::ecoinvent_activities
+
+ecoinvent_europages <- tiltIndicatorAfter::matches_mapper
+
+ecoinvent_inputs <- tiltIndicatorAfter::ecoinvent_inputs |>
+  select(
+    "input_activity_uuid_product_uuid",
+    "exchange_name",
+    "exchange_unit_name"
+  ) |>
+  distinct()
 
-# emissions_profile products
-ep_product <- read_csv("/path/emissions_profile_products_ecoinvent.csv")
+isic_tilt <- tiltIndicatorAfter::isic_tilt_mapper
 ```
 
-### Create Emissions_profile product and company level outputs
-
-The code takes the output of tiltIndicator and use it as an input for the
-tiltIndicatorAfter package.
+## Emissions profile
 
 ```{r}
-emissions_profile <- profile_emissions(ep_company,
-  ep_product,
+# Load data specific to this indicator
+emissions_profile_any_companies <- read_csv(toy_emissions_profile_any_companies())
+# FIXME User toy_emissions_profile_products_ecoinvent()
+# See https://github.com/2DegreesInvesting/tiltToyData/pull/12
+# https://github.com/2DegreesInvesting/tiltWorkflows/issues/9
+emissions_profile_products <- read_csv(toy_emissions_profile_products())
+
+# Create results at product and company level
+emissions_profile <- profile_emissions(
+  emissions_profile_any_companies,
+  emissions_profile_products,
   europages_companies = europages_campanies,
-  ecoinvent_activities = ei_activities_ov,
-  ecoinvent_europages = mapper_ep_ei,
-  isic_tilt = isic_4digit_name,
+  ecoinvent_activities = ecoinvent_activities,
+  ecoinvent_europages = ecoinvent_europages,
+  isic_tilt = isic_tilt,
   low_threshold = 1 / 3,
   high_threshold = 2 / 3
 )
 
-emissions_profile_at_product_level <- emissions_profile |> unnest_product()
-emissions_profile_at_company_level <- emissions_profile |> unnest_company()
-```
+emissions_profile_at_product_level <- emissions_profile |> 
+  unnest_product()
+emissions_profile_at_product_level
```

Comment:
This prints the output. It's important so that the user knows what to expect without needing to run this file.